webpack.base.config.babel.js

```diff
@@ -30,7 +30,7 @@ import MarkdownItImplicitFigures from 'markdown-it-implicit-figures';
 ///////////////
 const buildDir = 'dist';
 const publicPath = '/';
-export const lessonSrc = '../oppgaver/src';
+export const lessonSrc = process.platform === 'win32' ? path.basename('../oppgaver/src') : '../oppgaver/src';
```

Comment:
Nei, det er bare er quick fix for å fjerne alle feilmeldingene når du kjører `npm start` i Windows.